src/types/interfaces.ts

```diff
@@ -548,6 +566,12 @@ export interface IUpdateOutgoingMedia extends ISdkMediaDeviceIds {
 export interface IAcceptSessionRequest extends ISdkMediaDeviceIds {
   conversationId: string;
 
+  /** 
+   * this is optional, however if you have the sdk configured for screen recording
+   * you will want to specify this since the sessions share a conversationId.
+   */
+  sessionType?: SessionTypes;
```

Comment:
actually I don't think we do. You can't end screen recording sessions client-side or at least we want to obscure/limit that as much as possible. When you call endSession, it will ignore screen recording sessions. You *could* end the session if you did a force-terminate, but it *should* be the servers responsibility to end sessions.